x-pack/plugins/security_solution/public/entity_analytics/components/risk_information/index.tsx

```diff
@@ -52,18 +61,37 @@ const getTableColumns = (riskEntity?: RiskScoreEntity): Array<EuiBasicTableColum
   },
 ];
 
-interface TableItem {
-  range?: string;
-  level: RiskSeverity;
-}
-
-const tableItems: TableItem[] = [
+const riskLevelTableItems: RiskLevelTableItem[] = [
   { level: RiskSeverity.critical, range: i18n.CRITICAL_RISK_DESCRIPTION },
   { level: RiskSeverity.high, range: '70 - 90 ' },
   { level: RiskSeverity.moderate, range: '40 - 70' },
   { level: RiskSeverity.low, range: '20 - 40' },
   { level: RiskSeverity.unknown, range: i18n.UNKNOWN_RISK_DESCRIPTION },
 ];
+
+interface CriticalityLevelTableItem {
+  level: CriticalityLevel;
+  weight: number;
+}
+
+const criticalityLevelTableItems: CriticalityLevelTableItem[] = (
+  Object.keys(CriticalityModifiers) as CriticalityLevel[]
```

Comment:
I wonder if we can avoid this `as` assertion 🤔 
I believe the `fp-ts/Record` module has a `keys` function that preserves the key type